Rakefile

```diff
@@ -6,7 +6,7 @@ require 'rubocop/rake_task'
 
 Rails.application.load_tasks
 RuboCop::RakeTask.new
-task default: [:create_reports_dir, :rubocop, 'cucumber:html', 'brakeman:run', 'bundle_audit:run',
+task default: [:create_reports_dir, :rubocop, 'brakeman:run', 'bundle_audit:run',
```

Comment:
Do we want to readd cucumber tests to the basic rake run despite them not working yet?

Just checking, I assumed we wanted to wait until they were passing before doing so, but never got confirmation on that.